bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/JettyServer.java

```diff
@@ -33,12 +35,14 @@ public JettyServer(Supplier<JettyHandler> handler) {
 		this.handler = handler;
 	}
 
-	public void launch(Port port) throws JettyException {
+	public void launch(BindAddress bindAddress, Port port) throws JettyException {
```

Comment:
I agree on concise naming and being as short as possible. A convention should help guide and not enforce a inferior choice of name. In this specific case the variable name is the same as the command line argument, e.g. -server.port<->port and -server.bindaddress<->bindAddress. 

First, the command line argument must not be shortened to -server.address because then the naming is unclear. One could assume it is an address the server should connect to over the network rather then bind to it itself and listen for connections on the binding address. 

Then the variable name has to match the command line argument to make it clear that the argument's value ends up in the variable.

Second, just the word bind is out of scope, the variable does not describe a data strucutre that holds a bind to a network interface.

Therefore, please reconsider the name again.

Reply:
please pay attention to [my comment above](https://github.com/eclipse-passage/passage/pull/1102#discussion_r909355006)